CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Unreleased features
+
+(None yet)
+
+# 0.3.0 (2024-08-13)
+
+New features:
+
+- Integrated libp2p for transport encryption and authentication of peers.
+- Added a password protection – peers must specify the same `--secret` to connect.
+- You can now provide default startup arguments in `.ethersync/config`.
+- When editors send
+
+Bug fixes:
+
+- The daemon is now more resilient against misbehaving peers and editors, and won't crash as easily.
+
+# 0.2.2 (2024-08-01)
+
+New features:
+
+- Persist the CRDT to `.ethersync/doc`, so that peers can run independently, and sync back later.
+- When restarting the daemon, calculate file diffs since the daemon was last online, and apply them to the CRDT.
+- Dynamically create files when they are opened.
+- When files are deleted, delete them for all peers.
+- Sandbox file I/O to be restricted into the shared project folder.
```

Comment:
The editor plugin can now send JSON-RPC requests and get feedback on whether the intended operation worked or not (#74). Or less "internal": nvim plugin can now process feedback on whether opening a file was successful (#87)

btw, we could also add PRs to this changelog? 🤔 probably overkill to backdate, but going forward... there's probably an app for that? :D (i.e. a tool to scrape our git(hub) log and generate a summary)

Reply:
> there's probably an app for that

Yeah, like https://github.com/MarcoIeni/release-plz :D (Which follows https://www.conventionalcommits.org/en/v1.0.0/)